pkg/unikontainers/hypervisors/firecracker.go

```diff
@@ -88,9 +89,18 @@ func (fc *Firecracker) Execve(args ExecArgs) error {
 	}
 
 	// VM config for Firecracker
+	mem := uint(256)
```

Comment:
Could we have this as constant for all hypervisors (e.g. in the vmm.go file as a default memory).

Reply:
Yes, that could be helpful. I changed it.